Remotion/Web/Core/ExecutionEngine/WxeFunctionStateManager.cs

```diff
@@ -172,7 +174,7 @@ public WxeFunctionState GetItem (string functionToken)
       WxeFunctionState functionState;
       lock (_lockObject)
       {
-        functionState = (WxeFunctionState)_session[GetSessionKeyForFunctionState(functionToken)];
+        functionState = (WxeFunctionState)_session[GetSessionKeyForFunctionState(functionToken)]!;
```

Comment:
@MichaelKetting This is the cheap fix for the nullability problem but we could also backport https://re-motion.atlassian.net/browse/RM-9089 if you want. I don't know if we want that though, so for now its just the nullability override.

Reply:
Cheap fix it is